### ISSUE TITLE
building telemetry plugin in non-Windows environments

### ISSUE THREAD

**Issue by RootlessAgrarian** [Funbit/ets2-telemetry-server#103]
Has anyone built a telemetry plugin for environments other than Windows, i.e. OSX, Linux?  I am about to give this a try but would appreciate any dire warnings, war stories, etc. from anyone who has been down this path before me.

**Comment by Funbit** [comment]
I'm not sure that I understood you right, but the source of the telemetry plugin is mentioned in the readme: https://github.com/Funbit/ets2-sdk-plugin

**Comment by RootlessAgrarian** [comment]
Yes thank you Funbit, I edited my post since you read it, sorry.  I did find the SDK source from SCS and am starting to have a look, just wondering if anyone else has been here before me (i.e. ***ix environment).
